### ISSUE TITLE
Training with no kaldi

### ISSUE THREAD

**Issue by asmodaay** [iiscleap/NeuralPlda#2]
Hello, ty for code! Can model works without `LoadPldaParamsFromKaldi`? I train NeuralPLDA with  embeddings from another model and it overfitted on 3-4 epoch. So, how many pairs should i have to train it?

**Comment by iiscleap** [comment]
Hi! If you wanted to use the `SoftCdet` cost function, I would suggest to initialize with a pre-trained kaldi PLDA model. If you do not have a pretrained PLDA model, I would suggest to use `crossentropy` as the loss function and skip the `LoadPldaParamsFromKaldi` part.

SoftCdet is very bumpy cost function with many sub-optimal minima and saddle points. If the model is not initialized well, it will get stuck in such minimas or saddle points. Once the model has reached a good point in terms of crossentropy, you could switch over to SoftCdet.

We have observed that the model trains for as few as a few hundred thousand pairs (trials) such as the SRE 18 dev trials, upto 30 million pairs generated from Voxceleb augmented dataset with 6 million utterances. If you have n utterances, the number of pairs can be upto n*(n-1)/2, but you may want to take care of things like the number of utterances per speaker, duration, etc while sampling the trials to avoid overfitting.

**Comment by asmodaay** [comment]
Ty for fast answering, i will try ur tips!
